config/client.go

```diff
@@ -199,12 +199,11 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 			props["CWSURL"] = *c.CloudSettings.CWSURL
 		}
 
-		if *license.Features.SharedChannels {
+		if license.SkuShortName == model.LicenseShortSkuProfessional || license.SkuShortName == model.LicenseShortSkuEnterprise {
+			*license.Features.SharedChannels = true
```

Comment:
It seems, instead of assignment, it should be part of the `if` condition check.

Reply:
Yeah that was my initial thought as well, but was unable to find how this was set up. I'm currently in the process of investigating this and will update this PR once it's ready again.